core/src/test/java/com/bobocode/bring/core/context/impl/BringApplicationContextHappyCasesTest.java

```diff
@@ -312,5 +312,21 @@ void shouldCreatePrototypeConfigurationBean() {
         assertThat(bean).isNotEqualTo(bean2);
         assertThat(bean.getUuid()).isNotEqualTo(bean2.getUuid());
     }
-    
+
+    @DisplayName("Should inject implementations of Interface to Field in Order")
+    @Test
+    void shouldInjectListOfInterfaceImplementationToFieldInOrder() {
+        // given
+        var bringApplicationContext = BringApplication.run(TEST_DATA_PACKAGE + ".listfieldinjector");
+
+        // when
+        var aBean = bringApplicationContext.getBean(com.bobocode.bring.testdata.di.positive.listfieldinjector.AField.class);
+        aBean.talking();
+
+        // then
+        assertThat(aBean).isNotNull();
+        assertThat(aBean.getList()).isNotNull();
+        assertThat(aBean.getList()).hasSize(2);
```

Comment:
Need to check that order is correct. Just need to manually get 0 and 1 and ensure that we respect the order.